recipes/boutiques/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "boutiques" %}
+{% set version = "0.5.27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/boutiques/boutiques/archive/{{ version }}.tar.gz
+  sha256: f01607d37208d45fec3c371f08ccd7f456eab859cc0ec6d5852938f87e7fef99
+
+build:
+  entry_points:
+    - bosh=boutiques.bosh:bosh
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - simplejson
+    - termcolor
+    - jsonschema
+    - tabulate
```

Comment:
I don't think these need to be installed here. Does it also work without them?

Reply:
It doesn't - I tried to build locally on a linux machine and it didn't work. The build fails when I omit even a single dependency.

Reply:
The upstream `setup.py` is misusing the `setup-requires` field:
https://github.com/boutiques/boutiques/blob/master/setup.py#L56

so all of these are needed.